lib/timers.js

```diff
@@ -345,6 +347,7 @@ Timeout.prototype.close = function() {
   } else {
     exports.unenroll(this);
   }
+  return this;
 };
```

Comment:
Probably can't rename it, maybe could document it, though I don't know if it has a use case. Isn't the timer API inherited from browsers?